src/plugins/intel_gpu/src/plugin/infer_request.cpp

```diff
@@ -634,33 +417,17 @@ InferRequest::InferRequest(const std::vector<std::shared_ptr<const ov::Node>>& i
 // ----------------------------------------------------------------------------------------- //
 void InferRequest::preprocess_notify() {
     m_graph->wait(Graph::Stage::PREPROC);
-    if (m_graph->GetMaxDynamicBatchSize() > 1) {
-        preprocess_dynamic();
-    } else {
-        execDataPreprocessing(_inputs, true);  // "true" stands for serial preprocessing in case of OpenMP
-    }
     m_graph->notify(Graph::Stage::PREPROC);
 }
 
-void InferRequest::preprocess() {
-    if (m_graph->GetMaxDynamicBatchSize() > 1) {
-        preprocess_dynamic();
-    } else {
-        execDataPreprocessing(_inputs, true);  // "true" stands for serial preprocessing in case of OpenMP
-    }
-}
+void InferRequest::preprocess() {}
```

Comment:
is this needed?

Reply:
It seems unnecessary anymore for new infer request impl. removed